zzzz_modular_occulus/code/game/jobs/job/security.dm

```diff
@@ -12,12 +12,31 @@
 		STAT_VIG = 25
 	)
 /datum/job/medspec
+	access = list(
+		access_security, access_brig, access_moebius, access_sec_doors, access_medspec, access_morgue,
+		access_maint_tunnels, access_medical_equip, access_engine, access_mailsorting, access_external_airlocks, access_eva
+	)
 	stat_modifiers = list(
 		STAT_BIO = 25,
-		STAT_TGH = 10,
-		STAT_VIG = 15
+		STAT_TGH = 5,
+		STAT_VIG = 20,
+		STAT_ROB = 5
 	)
+	description = "You are a highly trained medical specialist within Aegis. You thusly have a combination of medical and military training. You are not quite as knowledgeable as a civilian career doctor, not quite as much of a fighter as a dedicated Aegis operative.<br>\
+	<br>\
+	Within Aegis, you have the following roles<br>\
+	<br>\
+	1. Field Medic. <br>\
+	You are expected to serve directly behind the frontlines in a combat situation, treating and stabilising the wounded. You may need to perform emergency trauma surgery and are equipped to do so. You are not expected nor qualified to act as a civilian doctor. <br>\
+	<br>\
+	2. Fire Support.<br>\
+	Often you will be assigned alongside an operative or inspector as a partner. In those situations you may act as fire support for your partner. Try and minimize risk to yourself, as your skill set is invaluable and you are not able to help others while incapacitated or dead.<br>\
+	<br>\
+	3. Brig Support. <br>\
+	During quiet times, you are responsible for assisting the warden in the as a local medic and backup forensics specialist. You are also capable of supporting Inspectors by performing lab work alongside them and should ensure the well-being of prisoners under the care of the Gunnery Sergeant."
```

Comment:
There's a typo here. 

"...for assisting the warden **in the as a local** medic..."